specification/web/resource-manager/Microsoft.Web/2016-03-01/Diagnostics.json

```diff
@@ -0,0 +1,1532 @@
+          "operationId": "Diagnostics_ListSiteDetectors",
+          "parameters": [
+            {
+              "$ref": "#/parameters/resourceGroupNameParameter"
+            },
+            {
+              "name": "siteName",
+              "in": "path",
+              "description": "Site Name",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "name": "diagnosticCategory",
+              "in": "path",
+              "description": "Diagnostic Category",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "$ref": "#/parameters/subscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/apiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/DiagnosticDetectorCollection"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{siteName}/diagnostics/{diagnosticCategory}/detectors/{detectorName}": {
+        "get": {
+          "tags": [
+            "Diagnostics"
+          ],
+          "summary": "Get Detector",
+          "description": "Get Detector",
+          "operationId": "Diagnostics_GetSiteDetector",
+          "parameters": [
+            {
+              "$ref": "#/parameters/resourceGroupNameParameter"
+            },
+            {
+              "name": "siteName",
+              "in": "path",
+              "description": "Site Name",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "name": "diagnosticCategory",
+              "in": "path",
+              "description": "Diagnostic Category",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "name": "detectorName",
+              "in": "path",
+              "description": "Detector Name",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "$ref": "#/parameters/subscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/apiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/DiagnosticDetectorCollection"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{siteName}/diagnostics/{diagnosticCategory}/detectors/{detectorName}/execute": {
```

Comment:
Same comment w.r.t POST vs PUT as for Analysis

Reply:
same as above